src/System.Diagnostics.Debug/tests/DebuggerTests.cs

```diff
@@ -11,19 +11,35 @@ public class DebuggerTests
         [Fact]
         public void IsAttached()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            bool b = Debugger.IsAttached;
         }
 
         [Fact]
-        public void Launch()
+        public void Break()
         {
-            // TODO: Implement this when Debugger is properly implemented
+            if (!Debugger.IsAttached)
+                Debugger.Break();
```

Comment:
Calling Debugger.Break when a Debugger **is** attached will also mess things up. The test isn't particularly useful anyways; I'm removing it.